### ISSUE TITLE
Have Hubot repeat sender names

### ISSUE THREAD

**Issue by codymcowan** [cycomachead/hubot-group-alias#15]
In many chat applications, when using this tool a user will receive the notification from Hubot, and see only the message Hubot sends (which as currently designed does not include who sent the original message). Having Hubot repeat the name of the sender, as shown below, would provide more context to those receiving a Hubot notification.

Current:
----

`Alice> Hey @dev, there's a standup in 5 min.`
`Hubot> Hey @Bob @Eve, there's a standup in 5 min.`

Proposed
----
`Alice> Hey @dev, there's a standup in 5 min.`
`Hubot> Alice: Hey @Bob @Eve, there's a standup in 5 min.`

**Comment by cycomachead** [comment]
Yes, I think this makes sense. (It was once on one of my TODO lists...)

I don't have much time right now, but I would certainly welcome a PR. :)
